brianlong/www.validators.app
# 183882630 validator details vue

#### What's this PR do?
- rewride validator details page to vue

#### How should this be manually tested?
- run tests
- check if page looks similar to production
- check if all the links are working

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183882630)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
